control/server.py

```diff
@@ -144,9 +144,9 @@ def _start_spdk(self):
 
         # Get path and import SPDK's RPC modules
         spdk_path = self.config.get("spdk", "spdk_path")
-        sys.path.append(spdk_path)
+        sys.path.append(os.path.join(spdk_path, "spdk/python"))
```

Comment:
I am fine with this.

Reply:
> Could be done as a separate PR though not to delay this merge.

I think a separate PR is the only way to go here since it is a user impacting change.  We only have a couple now, but still -- one would need to change their config file after this lands.